src/viam/components/gripper/client.py

```diff
@@ -24,17 +25,23 @@ def __init__(self, name: str, channel: Channel):
         self.client = GripperServiceStub(channel)
         super().__init__(name)
 
-    async def open(self, *, timeout: Optional[float] = None):
-        request = OpenRequest(name=self.name)
+    async def open(self, *, timeout: Optional[float] = None, extra: Optional[Dict[str, Any]] = None):
```

Comment:
It's not strictly necessary in the language, but for consistency with the other components, let's have `extra` come BEFORE `timeout`:
```python3
async def open(self, *, extra: ..., timeout: ...):
```